x-pack/plugins/lens/public/shared_components/legend/legend_settings_popover.tsx

```diff
@@ -333,13 +333,9 @@ export const LegendSettingsPopover: React.FunctionComponent<LegendSettingsPopove
                 })}
                 data-test-subj="lens-legend-show-value"
                 showLabel={false}
-                checked={legendStats?.[0] === LegendStats.values}
+                checked={!!legendStats?.length}
```

Comment:
this is just temporary, will be changed very soon when legend statistics are introduced